app/Livewire/ProspectPipelineKanban.php

```diff
@@ -107,20 +108,22 @@ public function render()
         ]);
     }
 
-    public function moveProspect(Pipeline $pipeline, string $educatableId, $fromStage = '', $toStage = '', string $educatableType = ''): JsonResponse
+    public function moveProspect(Pipeline $pipeline, string $educatableId, $fromStage = '', $toStage = ''): JsonResponse
     {
         try {
+            $model = $pipeline->segment->model;
+
+            $educatableType = match ($model) {
+                SegmentModel::Prospect => 'prospect',
+                SegmentModel::Student => 'student',
+            };
```

Comment:
You can just use the class name directly instead of mapping it to `'student'` or `'prospect'`

```
$educatableType === Prospect::class
```